.github/workflows/black-pr.yml

```diff
@@ -45,14 +45,14 @@ jobs:
           content=$(</d/changes.txt)
           while read file
           do
-            if [ $(python -c "print('${file}'[-2:])") == "py" ]
+            if [[ ($(python -c "print('${file}'[-3:])") == ".py") && (-f ${file}) ]]
             then
               black --diff $file
             fi
           done </d/changes.txt
           while read file
           do
-            if [ $(python -c "print('${file}'[-2:])") == "py" ]
+            if [[ ($(python -c "print('${file}'[-3:])") == ".py") && (-f ${file}) ]]
```

Comment:
I have some reservations about using Python instead of Bash for this conditional, but its something we can come back and fix.

This works for the time being, and we have multiple PRs waiting for this one, so approved!